### DISCUSSION TITLE
Undefined variable $vocabulary

### DISCUSSION THREAD

**Discussion by icodealone** [jellydn/next-swagger-doc#832] [Q&A]
```
 ⨯ ./node_modules/swagger-ui-react/swagger-ui.css:2:2
Syntax error: postcss-simple-vars: <css input> Undefined variable $vocabulary

  1 | .models .json-schema-2020-12:not(.json-schema-2020-12--embedded) > .json-schema-2020-12-head .json-schema-2020-12__title:first-of-type {
> 2 |   font-size: 16px;
    |  ^
  3 | }
```
I"ve installed and followed instructions for NextJS13.  All works apart from the styling.  My build is throwing out this error and I'm not sure how to address it.   If I leave out the CSS import it works, but is obviously unstyled.

**Comment by jellydn** [comment]
> swagger-ui-react
This UI has a lot of open issues. Could you raise with them? https://github.com/swagger-api/swagger-ui/issues
